openpgp-dsm/src/lib.rs

```diff
@@ -373,14 +480,15 @@ pub fn generate_key(
         links: Some(links),
         ..Default::default()
     };
+    dsm_client.__update_sobject(
+        &subkey.uid()?, &link_update_req, "bind subkey to primary key".into()
+    )?;
 
-    http_client.update_sobject(&subkey.uid()?, &link_update_req)?;
-
-    info!("generate certificate");
     // Primary + sig, UserID + sig, subkey + sig
     let mut packets = Vec::<Packet>::with_capacity(6);
 
     // Self-sign primary key
+    info!("generate certificate - self-sign primary key");
```

Comment:
nit: can we somehow add a prefix for dsm specific logs to make filtering out easier?
like:
```rust
macro_rules! sq_dsm_log {
    ($log_type:path, $($log:tt)*) => {{
        $log_type ! ("sq-dsm log: {}", std::format!($($log)*));
    }};
}

// use like
sq_dsm_log!(info, "generate certificate - self-sign primary key");
```